build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/PerformanceTestPlugin.kt

```diff
@@ -303,7 +303,8 @@ fun Project.propertiesForPerformanceDb(): Map<String, String> {
     } else {
         selectStringProperties(
             PropertyNames.dbUrl,
-            PropertyNames.dbUsername
+            PropertyNames.dbUsername,
+            PropertyNames.dbPassword
```

Comment:
What does this change have to do with the rename?

Reply:
I needed/wanted to update the performance runtimes before I did the rename. So I needed the build to pick up the password I have in my `gradle.properties` file.